limit-children/functions/index.js

```diff
@@ -22,14 +22,15 @@ const MAX_LOG_COUNT = 5;
 
 // Removes siblings of the node that element that triggered the function if there are more than MAX_LOG_COUNT.
 // In this example we'll keep the max number of chat message history to MAX_LOG_COUNT.
-exports.truncate = functions.database.ref('/chat/{messageid}').onWrite(async (change) => {
-  const parentRef = change.after.ref.parent;
-  const snapshot = await parentRef.once('value');
-  if (snapshot.numChildren() >= MAX_LOG_COUNT) {
+exports.truncate = functions.database.ref('/chat').onWrite((change) => {
+  const parentRef = change.after.ref;
+  const snapshot = change.after
+
+  if (snapshot.numChildren() >= MAX_USERS) {
```

Comment:
What prompted the switch from `MAX_LOG_COUNT` to `MAX_USERS`? Looks like the comment in lines 23 and 24 show that this function was intentionally using `MAX_LOG_COUNT`